src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -500,6 +504,10 @@ $(function () {
           });
           var container = $('#affix > ul');
           container.scrollTop(container.scrollTop() + top - 100);
+
+          // calculate height of #affix
+          var parentHeight = $("#affix").parent().height();
+          $("#affix").css("height", parentHeight - contributionHeight - 50); // minus the height of margin top
         }
```

Comment:
I tried your sample page. The "Improve this Doc" button is not stable on Edge, is it caused by this code?